pkg/apis/tenancy/v1alpha1/helper/helper_test.go

```diff
@@ -143,6 +143,18 @@ func TestWorkspaceKey(t *testing.T) {
 		ws   string
 		want string
 	}{
+		{
+			name: "root ws",
+			org:  "",
+			ws:   RootCluster,
+			want: RootCluster,
+		},
+		{
+			name: "fake root ws",
+			org:  "myorg",
+			ws:   RootCluster,
+			want: "root:myorg#$#root",
+		},
```

Comment:
`ws:RootCluster` is not really RootCluster, but just a random workspace called "root".